modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/internal/ProcessCenterServerHolder.java

```diff
@@ -63,7 +63,8 @@ public static ProcessCenterServerHolder getInstance() {
     public void setRegistryService(RegistryService registrySvc) {
         this.registryService = registrySvc;
         updateArtifactPathPermissions();
-        deployAnalyticsDashboard();
+        deployAnalyticsDashboard("process-charts");
+        deployAnalyticsDashboard("user-analytics");
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7